src/plots/piePlot.ts

```diff
@@ -276,8 +375,9 @@ namespace Plottable.Plots {
       let dataset = this.datasets()[0];
       let data = this._getDataToDraw().get(dataset);
       let pie = d3.layout.pie().sort(null).value((d, i) => sectorValueAccessor(d, i, dataset))(data);
-      this._startAngles = pie.map((slice) => slice.startAngle);
-      this._endAngles = pie.map((slice) => slice.endAngle);
+      let factor = (this.endAngle() - this.startAngle()) / (2 * Math.PI);
```

Comment:
Looks like d3.layout.pie has something similar already: see https://github.com/d3/d3-3.x-api-reference/blob/master/Pie-Layout.md#startAngle . Can we take advantage of that?